src/common/components/floating-faq/index.scss

```diff
@@ -0,0 +1,297 @@
+      right: 1rem;
+
+      @include themify(day) {
+        background: $dusk-blue;
+        border: none;
+      }
+
+      @include themify(night) {
+        background: $dark-indigo;
+        border: none;
+      }
+    }
+  }
+}
+
+.faq-content-list {
+  height: 59vh;
+  margin-top: -2rem;
+  overflow-y: auto;
+  overflow-x: hidden;
+  position: absolute;
+  z-index: 999;
+  border-radius: 14px;
+  width: 435px;
+}
+
+.faq-content-list-item {
+  z-index: 999;
+  height: auto;
+  width: 405px;
+  margin-left: 1rem;
+  border-radius: 14px;
+  overflow-y: auto;
+
+  @include themify(day) {
+    background: $white-two;
+  }
+
+  @include themify(night) {
+    background: $dark;
+  }
+}
+
+.contact-us {
+  margin-top: 0.5rem;
+}
+
+.section-title {
+  font-weight: bold;
+  font-size: 1.5rem;
+  padding: 1rem 0.7rem;
+
+  @include themify(day) {
+    color: $charcoal-grey;
+  }
+
+  @include themify(night) {
+    color: $white-six;
+  }
+}
+
+.section {
+  cursor: pointer;
+  padding-right: 1rem;
+}
+
+.search-bar {
+  margin: 1rem 0;
+}
+
+.user-info {
+  padding-left: 4px;
+  color: $steel-grey;
+  font-size: 14px;
+
+  @include themify(day) {
+    color: $steel-grey;
+  }
+
+  @include themify(night) {
+    color: $light-grey;
+  }
+}
+
+.username,
+.message {
+  padding: 0 2.5rem;
+}
+
+.faq-article {
+  display: flex;
+  margin: 1rem 0;
+  border-radius: 0.5rem;
+  padding: 0.6rem 1rem;
+  line-height: 1.5;
+  font-size: 14px;
+
+  @include themify(day) {
+    background: $white-three;
+    color: $dark-six;
+  }
+
+  @include themify(night) {
+    color: $white-five;
+    background: $dark-two;
+  }
+
+  .faq-image {
+    margin-right: 0.5rem;
+    position: relative;
+    top: -2px;
+  }
+}
+
+.submit-btn {
+  float: right;
+  margin: 0 2.4rem 1rem 0;
+}
+
+.help-content {
+  width: 405px;
+  padding: 0 2rem;
+}
+
+.card-body {
+  padding: 0 0 4rem !important;
+}
+
+@media screen and (max-width: $sm-break) and (min-width: 285px) {
+  .submit-btn {
+    margin: 0 10px 16px 0;
+  }
+
+  .faq-article {
+    margin: 0.5rem 1rem;
+    font-size: 0.8rem;
+  }
+
+  .username,
+  .message {
+    padding: 0 0.7rem;
+  }
+
+  .user-info {
+    padding: 0 1.3rem;
+    font-size: 12px;
+  }
+
+  .search-bar {
+    padding: 0 1rem;
+  }
+
+  .help-content {
+    width: 268px;
+    padding: 0 3px;
+  }
+
+  .section-title {
+    padding: 0.7rem 0.4rem;
+    font-size: 1rem;
+  }
+
+  .faq-content-list-item {
+    width: 268px;
+  }
+
+  .faq-content-list {
+    margin-top: -1.5rem;
+    height: 49vh;
+    width: 300px;
+  }
+
+  .faq-welcome-message {
+    padding: 2.7rem 0 0 1.5rem;
+    font-size: 1.2rem;
+  }
+
+  .faq-container {
+    height: 62vh;
+    width: 300px;
+  }
+}
+
+@media (max-width: 285px) {
+  .faq-container {
+    height: 72vh;
+    width: 84vw;
+  }
+
+  .faq-content-list {
+    height: 57.5vh;
+    width: 82vw;
+  }
+
+  .faq-welcome-message {
+    font-size: 19px;
+    padding: 2rem 0 0 0.8rem;
+  }
+
+  .section-title {
```

Comment:
@MTSaeed use custom class name or move under top class so it won't break other similar named classes/components

Reply:
Yes , @feruzm Updated.